tests/core/tools.js

```diff
@@ -855,6 +855,52 @@
 			assert.areSame( escapedSelector, '\\30 ', 'only-one-number selector' );
 		},
 
+		// (#681)
+		'test escapeCss - escaped colon in the css selector': function() {
+			var selector = 'abc:def';
```

Comment:
Please group all var declarations in unit tests - the same you have done in `tools.js`.